integration-templates/calendly/mocks/paginate/get/event-invitees/scheduled_events/29413570-8d98-4001-a340-38abdd2a5d39/invitees.json

```diff
@@ -0,0 +1,41 @@
+[
```

Comment:
do we really need all of that? Just one file is enough for testing I guess?

Reply:
Will keep the files as I already edited them heavily.